docs/zh/UserGuide/Server/Cluster Setup.md

```diff
@@ -69,6 +69,55 @@ or
 > nohup cluster\target\cluster-0.11.0-SNAPSHOT1\sbin\start-node.bat  -internal_meta_port 9005 -internal_data_port 40012 -cluster_rpc_port 55561
 > nohup cluster\target\cluster-0.11.0-SNAPSHOT2\sbin\start-node.bat  -internal_meta_port 9007 -internal_data_port 40014 -cluster_rpc_port 55562
 ```
+## n节点n-1副本伪分布式一键搭建脚本
```

Comment:
Where to set the n-1 copy? At present, the default copy is 2 copies. I do not see the logic to modify the number of copies in the script?

Reply:
NODE_NUM can be any number.

Reply:
> Besides, there[1] are scripts that can deploy pseudo-distributed or distributed easily, maybe we can combine both?
> [1] https://github.com/LebronAl/QuickDeployForIoTDBTesting

Hi, @neuyilan:
   I'll be so glad if you can contribute scripts at[1] to IoTDB ：）

Reply:
You may not understand me. The number of copies is `default_ replica_ num` parameter in iotdb-cluster.properties to control,  `NODE_NUM`  can not control the number of copies, it can only  control the number of nodes. For example, set NODE_NUM = 5, but the copy is still 2, not 4.